docs/sidebarsGenAI.ts

```diff
@@ -6,36 +6,35 @@ const sidebarsGenAI: SidebarsConfig = {
       type: 'doc',
       id: 'index',
       className: 'sidebar-top-level-category',
+      label: 'Overview',
     },
     {
-      type: 'category',
-      label: 'Getting Started',
+      type: 'html',
+      value: '<h4><b>Getting Started</b></h4>',
+      defaultStyle: true,
+    },
+    {
+      type: 'doc',
+      id: 'getting-started/connect-environment',
+      label: 'Set Up MLflow',
+      className: 'sidebar-top-level-category',
+    },
+    {
+      type: 'link',
+      href: '/genai/tracing/quickstart/python-openai',
+      label: 'Tracing GenAI Apps',
```

Comment:
```suggestion
      label: 'Tracing Quickstart',
```

I heard that the term "apps" is sometimes confusing ("I'm not building an application"). Maybe we don't need to mention it here.